src/io/matpower.jl

```diff
@@ -665,3 +667,419 @@ function merge_generic_data(data::Dict{String,Any})
     end
 end
 
+"Export a power flow case in matpower format"
+function export_matpower(io::IO, data::Dict{String,Any})
+
```

Comment:
In stead of doing the mixed-unit conversion while writing the output, can you just call `make_mixed_units(data)` at this point?